added_values.go

```diff
@@ -0,0 +1,608 @@
+package goja
+
+import (
+	"hash/maphash"
+	"reflect"
+	"strconv"
+
+	"github.com/dop251/goja/unistring"
+)
+
+func (i valueNumber) MemUsage(ctx *MemUsageContext) (uint64, error) {
+	return NumberSize, nil
+}
+
+func (i valueNumber) ToInteger() int64 {
+	switch v := i.val.(type) {
+	case int:
```

Comment:
yes - since we keep the int type in otto, I kept the values in here as well. This is useful for bson types Long vs Int for example.

Reply:
When I was porting over goja from otto in baas, I didn't need those types so left them out. I can add them here, but ultimately it might be worth revisiting all of these added int types to the interface after we remove otto from baas.